app/views/smart_proxies/show.html.erb

```diff
@@ -49,12 +50,30 @@
         </div>
         <div class="row">
           <div class="col-md-4">
-            <%= _('Features') %>
+            <%= _('Active Features') %>
           </div>
           <div class="col-md-8">
             <%= @smart_proxy.features.to_sentence %> <%= refresh_proxy_icon(@smart_proxy, authorizer) %>
           </div>
         </div>
+        <div class="row">
+          <div class="col-md-4">
+            <%= _('Disabled Features') %>
+          </div>
+          <div class="col-md-8">
+          <% if authorized_via_my_scope(:smart_proxy, :view_smart_proxies, @smart_proxy) %>
+            <div class="tab-pane" id="failed-modules"
+              data-ajax-url="<%= failed_modules_smart_proxy_path(:smart_proxy_id => @smart_proxy) %>"><%= spinner %></div>
```

Comment:
this feature was started this way - I want to know that we have plugins
that were enabled but failed to start (and see the reason why).